python-flask-admin-portal-example/requirements.txt

```diff
@@ -1,14 +1,14 @@
 certifi==2021.5.30
 charset-normalizer==2.0.6
 click==8.0.1
-Flask==2.0.1
+Flask==2.0.3
```

Comment:
Different examples used different flask versions. I set them all to the same version.

Reply:
Any reason we can't bump to the latest Flask version? Also fine with keeping this out of scope.